client/state/followers/selectors.js

```diff
@@ -3,21 +3,29 @@
  */
 
 import { getSerializedQuery } from 'calypso/state/followers/utils';
+
 /**
  * Returns a list of followers for the given Query.
  *
  * @param  {object} state Global state tree
  * @param  {object} query Query paramaters by which the followers were fetched
  * @returns {object}       List of followers keyed by follower id
  */
-export function getFollowersForQuery( state, query ) {
+export const getFollowersForQuery = ( state, query ) => {
+	const serializedQuery = getSerializedQuery( query );
+	return (
+		state.followers?.queries[ serializedQuery ]?.ids.map(
```

Comment:
Is it possible that `followers` state tree won't exist? It seems it's not modular at this point? Or did you intend to make it modular?

Reply:
See my comment https://github.com/Automattic/wp-calypso/pull/47451#discussion_r526917994, will make this modular as part of this PR.